android/app/src/main/res/layout/item_track.xml

```diff
@@ -1,43 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="72dp"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:baselineAligned="false"
-    android:clickable="true"
-    tools:showIn="@layout/list_tracks"
-    android:focusable="true"
-    android:background="?attr/selectableItemBackground"
-    android:orientation="horizontal">
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    tools:showIn="@layout/list_tracks">
 
-    <LinearLayout
+    <android.support.v7.widget.CardView
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginTop="@dimen/item_margin"
-        android:layout_marginBottom="@dimen/item_margin"
-        android:layout_marginRight="@dimen/content_margin"
-        android:layout_marginLeft="@dimen/content_margin"
-        android:orientation="vertical"
-        android:gravity="center">
-
-        <TextView
-            android:id="@+id/track_title"
-            android:textSize="18sp"
-            android:textColor="#d3000000"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginBottom="10dp"
+        android:clickable="true"
+        app:cardBackgroundColor="#ffffff"
+        app:cardCornerRadius="2dp"
+        app:cardElevation="2dp">
 
+        <com.balysv.materialripple.MaterialRippleLayout
```

Comment:
Appcompat already allowes us to display ripples in cards, just set the following attribute to the LinearLayout inside Cardview
`android:background="?attr/selectableItemBackground"`
Also, for large areas, it is a good practice to make the ripple transparent rather than making it colored, so the method specified above would do.